build.sbt

```diff
@@ -157,7 +178,11 @@ val ignoredABIProblems = {
     exclude[IncompatibleResultTypeProblem]("com.twitter.chill.FieldAccessFinder.visitMethod"),
     exclude[MissingClassProblem]("com.twitter.chill.FieldAccessFinder"),
     exclude[MissingTypesProblem]("com.twitter.chill.FieldAccessFinder"),
-    exclude[DirectMissingMethodProblem]("com.twitter.chill.FieldAccessFinder.this")
+    exclude[DirectMissingMethodProblem]("com.twitter.chill.FieldAccessFinder.this"),
+    exclude[IncompatibleResultTypeProblem]("com.twitter.chill.Tuple1*Serializer.read"),
+    exclude[IncompatibleMethTypeProblem]("com.twitter.chill.Tuple1*Serializer.write"),
+    exclude[IncompatibleResultTypeProblem]("com.twitter.chill.Tuple2*Serializer.read"),
+    exclude[IncompatibleMethTypeProblem]("com.twitter.chill.Tuple2*Serializer.write")
```

Comment:
these require a version bump to `0.10.0` before we publish next. Can we not break compatibility?

Reply:
yeah that's the sad part ... I couldn't find a way to not break it unfortunately. I know it's kinda of annoying to have this version bump due to a minor scala version update; the bright side is that this should give us better compatibility for future versions since we don't reference internal classes directly,

Reply:
@johnynek do you think it's better to bump version in a separate PR?